README.md

```diff
@@ -51,15 +54,29 @@ will be fired when the animation is complete. See props below.
 | containerStyle  | object        | Yes       | See PinInput.js      | Style applied to PINS container.                                                        |
 | pinStyle        | object        | Yes       | See PinInput.js      | Style applied to each circle PIN.                                                       |
 | pinActiveStyle  | object        | Yes       | See PinInput.js      | Style applied to each circle PIN when it is active.                                     |
+| maxNumberOfLargePins| number    | Yes       |                      | The max number of large pins.                                                           |
```

Comment:
Let's change this here to `numberOfPinsMaximum` and let's change the description "The max number of large pins." to something which explains this feature more. i.e. "This property is used to set a maximum number of pins to show. If specified, then the pins will have an animation effect, as showcased in the demo above".